### PR TITLE
OVS - 3.0.1 changes

### PR DESCRIPTION
**Pull request by palatsangeetha**
Added cursor to vesselSchedules
Added location discriminator
Added additionalProperties:false to UNLocationLocation
Fixed the ErrorCodeMessage issue and added minItems to errors array.

### CODE REVIEW

**Review by HenrikHL — CHANGES_REQUESTED** [review]
You have to change the version number of the API as well (line 6 of the API)

**Review by HenrikHL — DISMISSED** [review]
I agree with the changes. Now it needs to be agreed if this is a Bug-fix release
If it is a bug-fix - should the issues in the ErrorObject be fixed as well?

File: ovs/v3/OVS_v3.0.1.yaml [hunk 1]
```diff
@@ -323,6 +341,12 @@ components:
             - $ref: '#/components/schemas/UNLocationLocation'
             - $ref: '#/components/schemas/FacilitySMDGLocation'
             - $ref: '#/components/schemas/AddressLocation'
+          discriminator:
```
Comment (emiliodcsa) [inline ovs/v3/OVS_v3.0.1.yaml:344]: Hi @palatsangeetha , why is this a change if already present in 3.0.0? 
image

File: ovs/v3/OVS_v3.0.1.yaml [hunk 2]
```diff
@@ -175,6 +190,12 @@ paths:
           in: header
           name: API-Version
           description: An API-Version header **MAY** be added to the request (optional); if added it **MUST** only contain **MAJOR** version. API-Version header **MUST** be aligned with the URI version.
+        - schema:
+            type: string
```
Comment (HenrikHL) [inline ovs/v3/OVS_v3.0.1.yaml:194]: please add the limitation we agreed on in the chat

**Review by HenrikHL — DISMISSED** [review]

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**